### ISSUE TITLE
Not possible to resize Launchers on touch screens

### ISSUE THREAD

**Issue by martaszmit** [jupyterlab/jupyterlab#16061]
## Description

I tried to use JupyterLab (browser version) on my tablet (Microsoft Surface Pro 9), but it is not possible to resize Launchers using touch.  It works fine using mouse. 

## Reproduce

Add multiple Launchers and try to resize it.
![image](https://github.com/jupyterlab/jupyterlab/assets/19773837/c474a7a5-cf19-40fd-b438-57cbc6d6846f)

## Expected behavior

Resizing should be possible on touch screens like it is when using mouse.

## Context

- Operating System and version: Windows 11 Home
- Browser and version: Chrome
- Jupyter version: https://jupyter.org/try-jupyter/lab/

**Comment by krassowski** [comment]
That's odd, since lumino 2.0 (included in JupyterLab 4.x) we are using pointer events rather than mouse events so it should no longer be the case.

>  https://jupyter.org/try-jupyter/lab/

As of today this is using JupyterLab 4.0.11.

**Comment by JasonWeill** [comment]
Curious whether this also occurs in Lab 4.1 or 4.2 alpha.

**Comment by martaszmit** [comment]
It works much better on my iPad, but doesn't work on my Windows devices. Not sure how about Android, because I don't have any device to test it.  It also doesn't work in browser developer tools while simulating touch.

**Comment by krassowski** [comment]
> It also doesn't work in browser developer tools while simulating touch.

Right, I can reproduce this. I see that there is something off with pointer events for handlers in lumino, but do not immediately see what is wrong. Maybe we should use https://developer.mozilla.org/en-US/docs/Web/API/Element/setPointerCapture API, not sure how much it would help though.

**Comment by martaszmit** [comment]
Hello,
I wanted to ask if there is any chance of solving this problem in the near future?

**Comment by krassowski** [comment]
If you can send a PR (or enable someone to send a PR) I can help with review :)
